packages/storybook-builder-parcel/preset.js

```diff
@@ -119,10 +119,15 @@ async function createParcel(options, isDev = false) {
         resolveFrom: __filename,
       },
     ],
-    defaultTargetOptions: {
-      distDir: options.outputDir,
-      publicUrl: "./",
-      shouldScopeHoist: isDev ? false : true,
-    },
+    targets: {
+      storybook: {
+        distDir: options.outputDir,
+        publicUrl: "./",
+        scopeHoist: isDev ? false : true,
+        engines: {
+          browsers: ['last 2 Chrome version', 'last 2 Safari versions', 'last 2 Edge version', 'last 2 Firefox versions']
```

Comment:
This may have been working around a parcel issue where we don't default to development targets or use browserslist for targets provided via the API